utility.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  pickUpReady: 'Pick-Up Ready',
```

Comment:
change this to `pickReady` that way you don't have to change each and every one to `pickUpReeady`